platform/tron/client.go

```diff
@@ -11,66 +10,57 @@ import (
 )
 
 type Client struct {
-	HTTPClient *http.Client
-	BaseURL    string
+	Request blockatlas.Request
+	BaseURL string
 }
 
-func (c *Client) GetTxsOfAddress(address string) ([]Tx, error) {
-	uri := fmt.Sprintf("%s/accounts/%s/transactions?%s",
-		c.BaseURL,
-		url.PathEscape(address),
-		url.Values{
-			"only_confirmed": {"true"},
-			"limit":          {strconv.Itoa(blockatlas.TxPerPage)},
-		}.Encode())
-	httpRes, err := c.HTTPClient.Get(uri)
-	if err != nil {
-		logger.Error(err, "Tron: Failed to get transactions")
-		return nil, blockatlas.ErrSourceConn
+func InitClient(BaseURL string) Client {
+	return Client{
+		BaseURL: BaseURL,
+		Request: blockatlas.Request{
+			HttpClient: http.DefaultClient,
+			ErrorHandler: func(res *http.Response, uri string) error {
```

Comment:
We can create a default error handler to avoid the creation of a dummy function in every client doesn't implement this. Like the `http.DefaultClient`

Reply:
Added an issue for it: https://github.com/trustwallet/blockatlas/issues/343